code/game/machinery/computer/communications.dm

```diff
@@ -273,6 +273,16 @@
 			src.emagged = 0
 			setMenuState(usr,COMM_SCREEN_MAIN)
 
+		if("RestartNanoMob")
+			if(mob_hunt_server)
+				var/loading_msg = pick("Respawning spawns", "Reticulating splines", "Flipping hat",
+									"Capturing all of them", "Fixing minor text issues", "Being the very best",
+									"Nerfing this", "Not communicating with playerbase", "Coding a ripoff in a 2D spaceman game")

```

Comment:
Tabs are intentionally inconsistent and based on user preference; you can't use them to align the start of one line with characters in the middle of another, because you never actually know how wide they're going to be.

In a case like this, you shouldn't be bothering to try and line them up. Just use one extra tab for the lines that are a continuation of the first.